components/tooltip/tooltip.js

```diff
@@ -892,19 +909,22 @@ class Tooltip extends RtlMixin(LitElement) {
 	}
 
 	async _showingChanged(newValue, dispatch) {
+		if (!this.isConnected) return;
```

Comment:
I'm not sure why this only started being necessary now, but `input-date-range` has two layers of validation tooltips, and the upper layer (which is suppressed if there are child validation errors), was showing itself and hiding the child error before then removing itself from the DOM to allow the child tooltip to show, but that one had already been hidden, so we ended up with none. This simply bails if the tooltip that called it is no longer connected to the DOM.